googlemediaframework/src/main/java/com/google/android/libraries/mediaframework/exoplayerextensions/ExoplayerWrapper.java

```diff
@@ -752,47 +668,6 @@ public void onBandwidthSample(int elapsedMs, long bytes, long bandwidthEstimate)
   }
 
   @Override
-  public void onLoadStarted(int sourceId,
-                            String formatId,
-                            int trigger,
-                            boolean isInitialization,
-                            int mediaStartTimeMs,
-                            int mediaEndTimeMs,
-                            long totalBytes) {
-    if (infoListener != null) {
-      infoListener.onLoadStarted(sourceId, formatId, trigger, isInitialization, mediaStartTimeMs,
-          mediaEndTimeMs, totalBytes);
-    }
-  }
-
-  @Override
-  public void onLoadCompleted(int sourceId, long bytesLoaded) {
-    if (infoListener != null) {
-      infoListener.onLoadCompleted(sourceId, bytesLoaded);
-    }
-  }
-
-  @Override
-  public void onLoadCanceled(int sourceId, long bytesLoaded) {
-    // Do nothing.
-  }
-
-  @Override
-  public void onDownstreamFormatChanged(int sourceId,
```

Comment:
Where there changes here, or just code reformatting? Please avoid large reformats unless there's some need for it.